data/json/items/ranged/pneumatic.json

```diff
@@ -36,5 +35,41 @@
       [ "underbarrel mount", 1 ]
     ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "BB": 150 } } ]
+  },
+  {
+    "id": "varmint_airgun",
+    "type": "GUN",
+    "looks_like": "ar15",
+    "reload_noise_volume": 10,
+    "symbol": "(",
+    "color": "brown",
+    "name": { "str": "varmint air rifle" },
+    "description": "A cheap varmint air rifle chambered in .177, suitable for hunting small game.",
+    "price": 25000,
+    "price_postapoc": 550,
+    "//": "Based on the Ruger Targis High Velocity",
+    "material": [ "plastic", "steel" ],
+    "flags": [ "NEVER_JAMS", "NON_FOULING" ],
+    "skill": "rifle",
+    "ammo": [ "airgun_pellet" ],
+    "weight": "3674 g",
+    "volume": "1500 ml",
+    "bashing": 9,
+    "to_hit": -1,
+    "range": 14,
+    "ranged_damage": { "damage_type": "stab", "amount": 2 },
```

Comment:
Why is there stab damage here?

Reply:
Hrm, you're probs right. I didn't check the "normal" damage for an air rifle or the average barrel length.